packages/functional-tests/tests/react-conversion/oauthResetPassword.spec.ts

```diff
@@ -218,7 +219,6 @@ test.describe('severity-1 #smoke', () => {
       pages: { login, resetPasswordReact },
       testAccountTracker,
     }) => {
-      test.fixme(true, 'Fix required as of 2023/07/18 (see FXA-8006).');
       // the PKCE button is broken at the moment, so for now navigate directly to the link.
       // PKCE button doesn't appear to work at the moment locally. Some sort of cors error
       // keeps getting in the way. Just go to link directly for now.
```

Comment:
Is this comment still true? If yes, I wonder if leaving the FXA-8006 reference in the comment would be helpful.